src/schema/rules/common.yaml

```
@@ -0,0 +1,276 @@
+---
+# This file describes rules for broad "standard" errors in the specification.
+
+# BIDS Validator Original Issue Code #26
+NiftiHeaderUnreadable:
+    code: NIFTI_HEADER_UNREADABLE
+    description: |
+        We were unable to parse header data from this NIfTI file.
+        Please ensure it is not corrupted or mislabeled.
+    level: error
+    validation: shallow
+    selectors:
+    - extension in [".nii", ".nii.gz"]
+
+# BIDS Validator Original Issue Code #27
+JsonInvalid:
+    code: JSON_INVALID
+    description: |
+        Not a valid JSON file.
+    level: error
+    validation: shallow
+    selectors:
+    - extension == ".json"
+
+# BIDS Validator Original Issue Code #28
+GzNotGzipped:
+    code: GZ_NOT_GZIPPED
+    description: |
+        This file ends in the .gz extension but is not actually gzipped.
+    level: error
+    validation: shallow
+    selectors:
+    - extension contains ".gz"
+
+# BIDS Validator Original Issue Code #30
+BvalMultipleRows:
+    code: BVAL_MULTIPLE_ROWS
+    description: |
+        .bval files should contain exactly one row of volumes.
+    level: error
+    validation: deep
+    selectors:
+    - extension == ".bval"
+
+# BIDS Validator Original Issue Code #31
+BvecNumberRows:
+    code: BVEC_NUMBER_ROWS
+    description: |
+        .bvec files should contain exactly three rows of volumes.
+    level: error
+    validation: deep
+    selectors:
+    - extension == ".bvec"
+
+# BIDS Validator Original Issue Code #36
+NiftiTooSmall:
+    code: NIFTI_TOO_SMALL
+    description: |
+        This file is too small to contain the minimal NIfTI header.
+    level: error
+    validation: shallow
+    selectors:
+    - extension in [".nii", ".nii.gz"]
```

Given that these are hints to the implementor, and not automatically run (probably), not critical but potentially useful.

```suggestion
    - extension in [".nii", ".nii.gz"]
    check:
    - file.contents.size < 348
```